crates/biome_configuration/src/formatter.rs

```diff
@@ -111,58 +107,6 @@ impl FromStr for FormatterConfiguration {
     }
 }
 
-pub fn to_format_settings(
```

Comment:
This logic is located in `crates/biome_service/src/settings.rs` with renaming to `to_override_format_settings`.